bare_metal_how_to.md

```diff
@@ -0,0 +1,176 @@
+# MVP - Bare metal provisioning
+
+## Preparation
+
+### Setup - Ironic server
+
+To be used as Ironic server the host's requirements are:
+
+1. CentOS Stream 9
+2. root access
+
+#### Install
+
+<!-- TODO: update after https://gitlab.cee.redhat.com/pnt-resource-hub/rhub-baremetal/-/merge_requests/11 -->
+
+Run playbook available on: https://gitlab.cee.redhat.com/pnt-resource-hub/rhub-baremetal
+
+```shell
+$ ansible-playbook -i inventory/dsal.yml playbooks/bifrost_setup.yml --extra-vars "target=setup<N>"
+```
+
+Get the password:
+
+```shell
+$ sudo grep -e password /etc/ironic/ironic.conf
+```
+
+#### Insertion on Resource Hub
+
+```python
+requests.post(
+    BASE_URL + 'handler',
+    json=dict(
+        name='setup1',
+        arch='x86_64',
+        location_id='<LOCATION_ID>',
+        user_name='admin',
+        password='...',
+        base_url='http://dell-r730-025.dsal.lab.eng.rdu2.redhat.com:6385',
```

Comment:
We shouldn't publicize RedHat host names in github.  Make something up like example.yourcompany.com